config/application.rb

```diff
@@ -51,13 +51,10 @@ class Application < Rails::Application
 
     config.guidance_url = "https://www.gov.uk/government/publications/additional-payments-for-teaching-eligibility-and-payment-details"
 
-    if ENV["LOGSTASH_HOST"].present?
-      tcp_logger = LogStashLogger.new(type: :tcp,
-                                      host: ENV.fetch("LOGSTASH_HOST"),
-                                      port: ENV.fetch("LOGSTASH_PORT"),
-                                      ssl_enable: true)
-
-      SemanticLogger.add_appender(logger: tcp_logger, level: :info, formatter: :json)
-    end
+    # Additional information which is passed in the logs for each request
+    # See https://rocketjob.github.io/semantic_logger/rails.html#named-tags
+    config.log_tags = {
```

Comment:
I wonder if it makes sense to put this in with the rest of the logstash config in that initializer?

Reply:
This is picked up by Semantic Logger in all contexts, not just when sending to Logstash. I debated putting it in there since that's where we use it, but actually this information would be useful no matter where logs might end up being sent so I figured it would work better being applied to across the board, even if Logstash wasn't initialised.